src/components/transactions/styles.ts

```diff
@@ -19,6 +19,19 @@ export const RightAlignContainer = styled.View`
   margin-right: 29px;
 `;
 
+export const RightAlignContainer = styled.View`
+  flex: 1;
+  width: 100%;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-content: center;
+`;
```

Comment:
Is this redundant styling to have the same component, differing only by its margin?